scripts/validate-gentx.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh
+REGEN_HOME="/tmp/regen$(date +%s)"
+RANDOM_KEY="randomregenvalidatorkey"
+CHAIN_ID=regen-1
+
+GENTX_FILE=$(find ./$CHAIN_ID/gentxs -iname "*.json")
+LEN_GENTX=$(echo ${#GENTX_FILE})
+
+# Gentx Start date
+start="2021-03-31 15:00:00Z"
+# Compute the seconds since epoch for start date
+stTime=$(date --date="$start" +%s)
+
+# Gentx End date
+end="2021-04-06 15:00:00Z"
```

Comment:
```suggestion
end="2021-04-09 15:00:00Z"
```

We've decided to push back the target mainnet launch to April 15th to give a full 2 weeks of candidate chain running.  I think we should update GenTx end date to April 9 (Friday), which gives a full Mon-Wed between gentx closure and launching mainnet.

Reply:
@clevinson in the blog post it is mentioned as 6th and I am still waiting for a call. So, keeping it as 6th for now and we can update later. Just want to be in-line with all other communications (announcements, blog post etc)

Reply:
Ok let's do that then!